src/documentation/pipelines/design-your-pipeline.md

```diff
@@ -31,7 +31,7 @@ The simplest pipelines represent a linear flow of operations, as shown in Figure
 </figure>
 Figure 1: A linear pipeline.
 
-However, your pipeline can be significantly more complex. A pipeline represents a [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) of steps. It can have multiple input sources, multiple output sinks, and its operations (transforms) can output multiple `PCollection`s. The following examples show some of the different shapes your pipeline can take.
+However, your pipeline can be significantly more complex. A pipeline represents a [Directed Acyclic Graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) of steps. It can have multiple input sources, multiple output sinks, and its operations (transforms) can read multiple `PCollection`s, then output multiple `PCollection`s. The following examples show some of the different shapes your pipeline can take.
```

Comment:
change to below line to avoid double-and ?
> It can have multiple input sources, multiple output sinks, its operations (transforms) can read multiple `PCollection`s, and output multiple `PCollection`s.